target/linux/ath79/dts/qca9557_engenius_esr1200.dts

```diff
@@ -73,12 +73,20 @@
 	status = "okay";
 };
 
+&nvmem {
+	calibration_art_5000: calibration@5000 {
+		reg = <0x5000 0x844>;
+	};
+};
+
 &wmac {
-	nvmem-cells = <&calibration_art_1000>;
-	nvmem-cell-names = "calibration";
+	nvmem-cells = <&calibration_art_1000>, <&macaddr_uboot_eth 1>;
```

Comment:
Why do you set it to `<&macaddr_uboot_eth 1>;` and not `<&macaddr_uboot_eth 0>? I think the old userspace code would set it to the macaddr_uboot_eth without offset.

Reply:
Ah not quite. PHYNBR changes with ath9k and ath10k. It's phy1 with the former and phy0 with the latter.

Reply:
ping @DragonBluep

Reply:
For the ath79 target, if PCIe NIC is ath10k based, the PCIe radio phy number is 0 because the ath10k driver will be loaded first.

Reply:
@DragonBluep for ath9k radios, is 501-ath9k_ahb_init.patch to blame for the difference?

Reply:
I think it may be because the kernel modules are loaded in alphabetical order.